apiserver/facades/controller/undertaker/register.go

```diff
@@ -24,11 +23,11 @@ func Register(registry facade.FacadeRegistry) {
 // newUndertakerFacade creates a new instance of the undertaker API.
 func newUndertakerFacade(ctx facade.ModelContext) (*UndertakerAPI, error) {
 	st := ctx.State()
-	model, err := st.Model()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
+
+	authFunc := common.AuthFuncForTag(names.NewModelTag(string(ctx.ModelUUID())))
```

Comment:
Can we use the .String method on `ModelUUID` instead?

Reply:
fixed.